src/test/java/gov/cancer/tests/crosscutting/Footer_Test.java

```diff
@@ -0,0 +1,544 @@
+package gov.cancer.tests.crosscutting;
+
+import java.util.Iterator;
+
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import gov.cancer.framework.ExcelDataReader;
+import gov.cancer.pageobject.crosscutting.FooterPage;
+import gov.cancer.tests.TestObjectBase;
+import gov.cancer.tests.TestRunner;
+
+/**
+ * Tests for pages with footer.
+ */
+public class Footer_Test extends TestObjectBase {
+
+  /**
+   * This method is checking if the Footer exists on the pages
+   * 
+   * @param path
+   *          Path of the page to check.
+   */
+  @Test(dataProvider = "getPageFooterPaths")
+  public void footerIsVisible(String path) {
+
+    TestRunner.run(FooterPage.class, path, (FooterPage page) -> {
+
+      Assert.assertTrue(page.isFooterVisible(), "Page Footer is visible.");
+
+    });
+
+  }
+
+  /**
+   * This method is checking if the header on the Footer exists and is correctly
+   * displayed on the pages
+   * 
+   * @param path
+   *          Path of the page to check.
+   */
+  @Test(dataProvider = "getPageFooterPaths")
+  public void verifyFooterHeader(String path) {
+
+    TestRunner.run(FooterPage.class, path, (FooterPage page) -> {
+
+      Assert.assertTrue(page.isFooterHeaderVisible(), "Footer Header is visible.");
+      Assert.assertTrue(page.getFooterHeader().getText().contains("National Cancer Institute"));
+
+    });
+
+  }
+
+  /**
+   * This method is checking if the FOLLOW US section exists on footer on the
+   * pages and correct links and labels are displayed
+   * 
+   * @param path
+   *          Path of the page to check.
+   */
+  @Test(dataProvider = "getPageFooterPaths")
+  public void verifyFooterFollowUs(String path) {
+
+    TestRunner.run(FooterPage.class, path, (FooterPage page) -> {
+
+      Assert.assertTrue(page.isFooterFollowUsVisible(), "FOLLOW US section in the Footer is displayed.");
+      Assert.assertTrue(page.getFooterFollowUs().getText().contains("FOLLOW US"));
+      Assert.assertTrue(page.getFooterFB().isDisplayed());
+      Assert
+          .assertTrue(page.getFooterFB().getAttribute("href").contains("https://www.cancer.gov/social-media#facebook"));
```

Comment:
This is a bad assertion.
1. This will pass, even if the href is something like "System.NullException - https://www.cancer.gov/social-media#facebook".  Don't use `contains()`.
1. Don't access methods on `WebElement`.  See the notes in the `FooterPage` class. This violates encapsulation.
1. Don't test against absolute URLs unless there is a requirement that they always be absolute URLs.  On the current system, these URLs are written as `/social-media#facebook` and this test would fail if run on one of the lower tiers.